RelatedPerson/RelatedPerson-FoetusToRyanneBoulder-Example.json

```diff
@@ -0,0 +1,21 @@
+{
+  "resourceType": "RelatedPerson",
+  "id":"RelatedPerson-FoetusToRyanneBoulder-Example",
+  "patient": {
+    "identifier": {
+      "system": "https://fhir.nhs.uk/Id/nhs-number",
+      "value": "9449307687"
+    }
+  },
+  "relationship": [
+    {
+      "coding": [
+        {
+          "system": "http://terminology.hl7.org/CodeSystem/v3-RoleCode",
+          "code": "PRN",
```

Comment:
Could you use a more specific code for mother as we need to differentiate mother from father